JiraFeatureCollector/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>com.capitalone.dashboard</groupId>
 	<artifactId>jira-feature-collector</artifactId>
 	<packaging>jar</packaging>
-	<version>01.00.00.01</version>
+	<version>01.00.01.01</version>
```

Comment:
this version scheme doesn't match the other plugins - can you update it to be semver style: x.x.x